packages/connectivity/connectivity/README.md

```diff
@@ -103,6 +103,34 @@ To request location authorization, make sure to add the following keys to your _
 * `NSLocationAlwaysAndWhenInUseUsageDescription` - describe why the app needs access to the user’s location information all the time (foreground and background). This is called _Privacy - Location Always and When In Use Usage Description_ in the visual editor.
 * `NSLocationWhenInUseUsageDescription` - describe why the app needs access to the user’s location information when the app is running in the foreground. This is called _Privacy - Location When In Use Usage Description_ in the visual editor.
 
+### No Location Permission 
+
+If you don't wish to set `NSLocationAlwaysAndWhenInUseUsageDescription` and `NSLocationWhenInUseUsageDescription` on iOS, you also have the option to disable the code that uses these permissions.  This may be useful if you don't wish to specify permissions that you have no intention of using and only want to use the plugin for checking the connection status of the device.  When disabled, the `getWifiName` and `getWifiBSSID` methods will throw a `MissingPluginException` error.
```

Comment:
Setting the usage description strings isn't what the developer cares about. What they care about is not using location permissions:

```suggestion
If you don't wish to use location permissions on iOS, you also have the option to disable the code that uses these permissions. This may be useful if you don't wish to specify permissions that you have no intention of using and only want to use the plugin for checking the connection status of the device. When disabled, the `getWifiName` and `getWifiBSSID` methods will throw a `MissingPluginException` error.
```

Reply:
Good point, updated